src/main/java/org/broadinstitute/hellbender/utils/pileup/PileupElement.java

```diff
@@ -152,15 +152,15 @@ public int getOffset() {
      * @return a base encoded as a byte
      */
     public byte getBase() {
-        return isDeletion() ? DELETION_BASE : read.getBases()[offset];
+        return isDeletion() ? DELETION_BASE : read.getBase(offset);
     }
 
     /**
      * Get the base quality score of the base at this aligned position on the genome
      * @return a phred-scaled quality score as a byte
      */
     public byte getQual() {
-        return isDeletion() ? DELETION_QUAL : read.getBaseQualities()[offset];
+        return isDeletion() ? DELETION_QUAL : (byte)read.getBaseQuality(offset);
```

Comment:
A question unrelated to this PR: why does `getBaseQuality()` return an int rather than a byte, given that `getBaseQualities()` returns a `byte[]`?